### PR TITLE
Add codes for peer to peer review

### PR DESCRIPTION
**Pull request by WilhelmK109**
This code is meant for peer to peer review

### CODE REVIEW

File: src/modules/activity.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+  }
+  localStorage.setItem('task', JSON.stringify(this.tasks));
+};
+
+const deleteTask = (tasks, index) => {
+  const newTasks = tasks.filter((task, innerIndex) => index !== innerIndex);
+  for (let i = 0; i < newTasks.length; i += 1) {
+    newTasks[i].index = tasks.length;
+  }
```
Comment (vangelif) [inline src/modules/activity.js]: I would advise you to review this for loop statement and make an attempt to replace with a forEach loop

File: src/modules/activity.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    if (task.isCompleted) {
+      span.removeAttribute('content-editable');
+      span.classList.add('complete');
+    } else {
+      span.setAttribute('content-editable', 'true');
+      parent.classList.remove('complete');
+    }
+  }
+  localStorage.setItem('task', JSON.stringify(this.tasks));
+};
```
Comment (vangelif) [inline src/modules/activity.js]: Here you have an if-else statement inside an if-else statement. Consider applying the KISS principle and review your code by making it more efficient.

File: src/modules/activity.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  saveItemToLocalStorage(newTasks);
+};
+
+const deleteCompletedTasks = (tasks) => {
+  const incomplete = tasks.filter((task) => task.completed !== true);
+  for (let i = 0; i < incomplete.length; i += 1) {
+    incomplete[i].index = i;
+  }
+  saveItemToLocalStorage(incomplete);
+};
```
Comment (vangelif) [inline src/modules/activity.js]: Here also, I would advise you to change the for loop statement with replace it a forEach loop instead.

**Review by vangelif — COMMENTED** [review]
Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy

You did a great job 💯 🥇 on build a project, however there are certain aspects, you need to work on, but you are almost there
 
### To Highlight 🥇 

- Great use of html fundamentals
- Displayed great use of Github

### Changes required 💯 

Please see comments below 👍🏼

File: src/modules/activity.js (hunk 1 above)
Comment (WilhelmK109) [reply src/modules/activity.js]: 👍

File: src/modules/activity.js (hunk 2 above)
Comment (WilhelmK109) [reply src/modules/activity.js]: Thank you

File: src/modules/activity.js (hunk 3 above)
Comment (WilhelmK109) [reply src/modules/activity.js]: 👍

**Pull request merged**